packages/shared-ux/chrome/navigation/src/ui/v2/components/navigation_group.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { createContext, useCallback, useMemo, useContext } from 'react';
+
+import { EuiButton } from '@elastic/eui';
+import { useInitNavnode } from '../use_init_navnode';
+import { NodeProps, RegisterFunction } from '../types';
+
+export const NavigationGroupContext = createContext<Context | undefined>(undefined);
+
+interface Context {
+  register: RegisterFunction;
+}
+
```

Comment:
same suggestion as above, and also to re-order the interface declaration:

```suggestion
interface NavigationGroupContext {
  register: RegisterFunction;
}

export const NavigationGroupContext = createContext<NavigationGroupContext | undefined>(undefined);
```